File: src/Author.cpp
#include <iostream>
#include <string>
#include "./User.cpp"

using namespace std;


class Author : public User {
    public:
        Author(string authorName) : User(authorName) {}

        void addBook()
        {
            cout << "Add Book successfully" << endl;
        }
        void removeBook()
        {
            cout << "Remove Book successfully" << endl;
        }
};

File: src/Author.js
// Assuming User.js has been similarly converted and is available
import User from "./User.js";

class Author extends User {
    constructor(authorName) {
        super(authorName);
    }

    addBook() {
        console.log("Add Book successfully");
    }

    removeBook() {
        console.log("Remove Book successfully");
    }
}

export default Author;
